packages/scripts/README.md

```diff
@@ -30,7 +30,8 @@ _Example:_
 		"check-licenses": "wp-scripts check-licenses",
 		"lint:css": "wp-scripts lint-style",
 		"lint:js": "wp-scripts lint-js",
-		"lint:md": "wp-scripts lint-md",
+		"lint-md:docs": "wp-scripts lint-md-docs",
```

Comment:
How do you feel about using: `lint:md-docs` and `lint:md-js` instead?

The idea here is that with libraries like https://www.npmjs.com/package/npm-run-all you can do the following in the `scripts` section:
```json
{
    "scripts": {
        "lint:js": "wp-scripts lint-js",
        "lint:md-docs": "wp-scripts lint-md-docs",
        "lint:md-js": "wp-scripts lint-md-js",
        "lint": "npm-s lint:*"
    }
}

Reply:
Makes sense, I was sure which way to go. Updated to this style :+1: